react/features/dynamic-branding/functions.any.js

```diff
@@ -13,6 +13,10 @@ export function extractFqnFromPath() {
     const parts = window.location.pathname.split('/');
     const len = parts.length;
 
+    if (parts.length === 2) {
+        return `/${parts[1]}`;
+    }
+
     return parts.length > 2 ? `${parts[len - 2]}/${parts[len - 1]}` : '';
```

Comment:
since you cannot have length < 2 , you can use the ternary : 

`return parts.length > 2 ? `${parts[len - 2]}/${parts[len - 1]}` : parts[1];`

Reply:
Done